go/engine/identify2_test.go

```diff
@@ -665,6 +665,33 @@ func TestIdentify2NoSigchain(t *testing.T) {
 	}
 }
 
+// TestIdentifyDeleted tests that identifying a deleted user gives the correct
+// error
+func TestIdentify2Deleted(t *testing.T) {
+	tc := SetupEngineTest(t, "Identify2Deleted")
+	defer tc.Cleanup()
+
+	u := CreateAndSignupFakeUserWithUser(tc, "login")
+	userId := u.User.GetUID()
+	DeleteAccount(tc, u)
+
```

Comment:
Hmm I did this, no bueno. Figuring out how to get it to output the hang, `kill -ABRT` gives me useless runtime-level output.